hikari/api/rest.py

```diff
@@ -1346,10 +1346,10 @@ async def fetch_webhook(
         Parameters
         ----------
         webhook : hikari.snowflakes.SnowflakeishOr[hikari.webhooks.Webhook]
-            The webhook to fetch. This may be a `hikari.webhooks.Webhook`
-            or the ID of an existing webhook.
+            The webhook to fetch. This may be a webhook object or the ID
```

Comment:
Could these bits not follow a more brief less repetitive format along the lines of "the object or ID of an existing webook"